modules/340-monitoring-ping/images/monitoring-ping/werf.inc.yaml

```diff
@@ -20,14 +37,15 @@ shell:
 ---
 image: {{ .ModuleName }}/{{ .ImageName }}
 fromImage: common/python-static
-git:
-- add: /{{ $.ModulePath }}modules/340-{{ $.ModuleName }}/images/{{ $.ImageName }}/
-  to: /src
-  stageDependencies:
-    install:
-    - '**/*'
+mount:
+- from: tmp_dir
```

Comment:
What the tmp_dir ?

Reply:
This is a temporary folder for the duration of the build. Its contents are not included in the image. This addition speeds up the build because less unnecessary information is crammed into the registry. 
It may be worth making generalized include for go, python and node.